test/Base_Tests/src/Network/Http_Spec.enso

```diff
@@ -10,7 +10,7 @@ import Standard.Base.Network.HTTP.Request_Body.Request_Body
 import Standard.Base.Network.HTTP.Request_Error
 import Standard.Base.Network.Proxy.Proxy
 import Standard.Base.Runtime.Context
-from Standard.Base.Network.HTTP import _resolve_headers, get_follow_redirects, get_proxy, get_timeout, get_version
+from Standard.Base.Network.HTTP import _resolve_headers, _get_follow_redirects, _get_proxy, _get_timeout, _get_version
```

Comment:
Wasn't the convention for `_` was to indicate module private methods?

Not opposed to using `_` for 'internal method that is exported for test purposes', just making sure we have a clear guideline on the naming.

Also, I guess if we want to do this proper, we could mark these methods as `private` and move these few test cases into `Base_Internal_Tests` that can load private modules.

But ofc happy to proceed as-is for now.

Reply:
I'm hoping we will get tests inside the same project soon and then these would be properly private.